github/attestations.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2024 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"encoding/json"
+)
+
+type Attestation struct {
```

Comment:
All exported structs need Godoc-styled comments for the auto-generated docs with complete sentences and punctuation.

Reply:
Ok, I'll add some.